### ISSUE TITLE
hestiaUI_font-NotoSans hestiaHUGO's Guide is Not Aligning to Goal

### ISSUE THREAD

**Issue by CoryGalyna** [ZORALab/Hestia#61]
# Description
```
Please provide a short description of what you have encountered below.
```
From what @hollowaykeanho told me about `hestiaHUGO` branding direction, it appears this page https://hestia.zoralab.com/en/docs/hestiaui/font-notosans_hestiaui/ is not guiding the user to install as it is. This... is a bug.

It won't be good for those without rich experience using Go.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
Guide should be:

```
...

staticDir = [
        "static",
        "themes/hestiaHUGO/fonts/hestiaUI_font-NotoSans",
]

...
```

# Current Behavior
```
Please specify the current behavior of software below.
```
Guide currently is:

```
...

staticDir = [
        "static",
        "themes/Hestia/fonts/hestiaUI_font-NotoSans",
]

...
```

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```
1. See https://hestia.zoralab.com/en/docs/hestiaui/font-notosans_hestiaui/

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Accepted. Feature is not broken. It's a documentation problem.

**Comment by hollowaykeanho** [comment]
Fixed in 6020bca888b2305398233a65d0a91666300a1151

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
